src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Yeah specification is not always "original specification", it's more like "specification that would yield the same result". How do you know there could be more than one counterparty in a single payment, is that mentioned in the docs?